src/core/client/framework/lib/bootstrap/TalkContext.tsx

```diff
@@ -1,14 +1,19 @@
 import { LocalizationProvider } from "fluent-react/compat";
 import { MessageContext } from "fluent/compat";
 import React, { StatelessComponent } from "react";
+import { Formatter } from "react-timeago";
 import { Environment } from "relay-runtime";
+import { UIContext } from "talk-ui/components";
 
 export interface TalkContext {
   // relayEnvironment for our relay framework.
   relayEnvironment: Environment;
 
   // localMessages for our i18n framework.
   localeMessages: MessageContext[];
+
+  // formatter for timeago.
+  timeagoFormatter: Formatter | null;
```

Comment:
Why not use:

```typescript
timeagoFormatter?: Formatter;
```

instead?

Reply:
I didn't want this to be optional, but to explicitly set to null in special cases e.g. for testing. Normally this is required.

Reply:
Thought about it again and made it optional :-P